### ISSUE TITLE
i2c should be disabled for BOARD=arduino_101

### ISSUE THREAD

**Issue by whbruce** [intel-iot-devkit/zmraa#45]
arduino_101 board definition has an i2c entry. I think it should be removed as it is not pinned out to Arduino shield so cannot be used.

**Comment by jontrulson** [comment]
Well, it (I2C) does work on the ARC core (only)... Am I missing something?

**Comment by whbruce** [comment]
ARC core means BOARD=arduino_101_sss. My issue is for BOARD=arduino_101 and reflects your "ARC core only" comment.

**Comment by jontrulson** [comment]
Ah, understood.

**Comment by Propanu** [comment]
Quick note here, it's exposed on the tinyTILE.

**Comment by malikabhi05** [comment]
I think the tinytile is implemented as a separate board altogether in Zephyr now. If it indeed has a separate board target in Zephyr, I think we can remove the i2c entry in the arduino_101 board definition and maybe create a separate definition for the tinytile.

**Comment by whbruce** [comment]
I agree with @malikabhi05, tinytile should be a separate board. I'm working on a101 right now, so will remove i2c entry.
